sample-plugin/pom.xml

```diff
@@ -14,10 +14,9 @@
     <properties>
         <revision>7</revision>
         <changelist>-SNAPSHOT</changelist>
-        <bom>2.190.x</bom>
-        <jenkins.version>2.190.3</jenkins.version>
+        <bom>2.222.x</bom>
+        <jenkins.version>2.222.1</jenkins.version>
         <java.level>8</java.level>
-        <slf4jVersion>1.7.26</slf4jVersion> <!-- TODO when dropping 2.150.x, reapply 77183ce09cce0e75e5509b211e7cd8ae2774a292 to get 4.x parent -->
```

Comment:
> Blocked on jenkinsci/workflow-cps-global-lib-plugin#93 currently

Does 77183ce09cce0e75e5509b211e7cd8ae2774a292 not suffice? I think we can drop 2.150.x now: https://github.com/jenkinsci/bom/#lts-lines

Reply:
Yeah that would work

Reply:
So, let us add 2.204.x and 2.222.x and drop 2.150.x. I _think_ there is a core BOM for 2.164.x so we are good there.

Reply:
(I would of course cut a final plugin BOM release including 2.150.x before merging this. Right now I am just gathering any easy PRs which just need rebuilds against `master`.)